lib/util.js

```diff
@@ -29,13 +29,21 @@ function rp (options) {
     });
 }
 
-function fetch_html(uri) {
+function fetch_html(uri, encoding=null) {
     uri = encode_uri(uri);
 
-    return rp({
+    const options = {
         uri: uri,
         gzip: true,
-    }).then(res => {
+    };
+
+    // Allow user to override encoding for instances where a site isn't giving
+    // the proper content-type
+    if (encoding) {
+        options.encoding = encoding;
+    }
```

Comment:
no, because `encoding: null` is meaningful to request and not what we want by default